missing checksum for aerospike-client-c-libuv-4.6.20.el6.x86_64.tgz

With the update from `3.16.2` to `3.16.3` an issue with a missing checksum appeared. IMO the commit [acc57fb](https://github.com/aerospike/aerospike-client-nodejs/commit/acc57fb0974838e92d241633a963bc29c3b47b87) is the issue here because the checksum for `el6.x86_64` got dropped.

working:
- OS: `Linux ** 5.8.0-43-generic #49~20.04.1-Ubuntu SMP Fri Feb 5 09:57:56 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux`
- Node.js: `12.20.2`
- Aerospike: `3.16.2`

working:
- OS: `Linux ip-**-**-**-**.eu-central-1.compute.internal 4.14.209-160.339.amzn2.x86_64 #1 SMP Wed Dec 16 22:44:04 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux`
- Node.js: `12.20.2`
- Aerospike: `3.16.2`

working:
- OS: `Linux ** 5.8.0-43-generic #49~20.04.1-Ubuntu SMP Fri Feb 5 09:57:56 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux`
- Node.js: `12.20.2`
- Aerospike: `3.16.3`

**failing**:
- OS: `Linux ip-**-**-**-**.eu-central-1.compute.internal 4.14.209-160.339.amzn2.x86_64 #1 SMP Wed Dec 16 22:44:04 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux`
- Node.js: `12.20.2`
- Aerospike: `3.16.3`

logs:
```
[ec2-user@ip-**-**-**-** reporting-cli]$ npm install

> aerospike@3.16.3 install /home/ec2-user/reporting-cli/node_modules/aerospike
> node-gyp rebuild

make: Entering directory `/home/ec2-user/reporting-cli/node_modules/aerospike/build'
  TOUCH ebf8c619d512e7a1a724a0ecb06ec5eec50df7dc.intermediate
  ACTION binding_gyp_aerospike_client_c_target_Installing_Aerospike_C_Client_dependency ebf8c619d512e7a1a724a0ecb06ec5eec50df7dc.intermediate
info: downloading 'https://artifacts.aerospike.com/aerospike-client-c/4.6.20/aerospike-client-c-libuv-4.6.20.el6.x86_64.tgz' to 'aerospike-client-c-libuv-4.6.20.el6.x86_64.tgz'
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  179k    0  179k    0     0  81916      0 --:--:--  0:00:02 --:--:-- 81916
grep aerospike-client-c-libuv-4.6.20.el6.x86_64.tgz /home/ec2-user/reporting-cli/node_modules/aerospike/aerospike-client-c.sha256
scripts/aerospike-client-c.sh: line 66: [: 0bb3a4186782b47e99b9e8bc73747720d7ad7e3b06e3fc129664f2a70a1189fa: unary operator expected
error: invalid checksum for 'aerospike-client-c-libuv-4.6.20.el6.x86_64.tgz'
make: *** [ebf8c619d512e7a1a724a0ecb06ec5eec50df7dc.intermediate] Error 1
rm ebf8c619d512e7a1a724a0ecb06ec5eec50df7dc.intermediate
make: Leaving directory `/home/ec2-user/reporting-cli/node_modules/aerospike/build'
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/home/ec2-user/.nvm/versions/node/v12.20.2/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:194:23)
gyp ERR! stack     at ChildProcess.emit (events.js:314:20)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:276:12)
gyp ERR! System Linux 4.14.209-160.339.amzn2.x86_64
gyp ERR! command "/home/ec2-user/.nvm/versions/node/v12.20.2/bin/node" "/home/ec2-user/.nvm/versions/node/v12.20.2/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /home/ec2-user/reporting-cli/node_modules/aerospike
gyp ERR! node -v v12.20.2
gyp ERR! node-gyp -v v5.1.0
gyp ERR! not ok 
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.3.1 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN services-reporting-cli@1.0.1 No repository field.
npm WARN services-reporting-cli@1.0.1 No license field.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! aerospike@3.16.3 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the aerospike@3.16.3 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ec2-user/.npm/_logs/2021-02-19T07_40_27_214Z-debug.log
```

--- a/scripts/aerospike-client-c.sh
+++ b/scripts/aerospike-client-c.sh
@@ -177,8 +177,8 @@ if [ $DOWNLOAD ] && [ $DOWNLOAD == 1 ]; then
           DISTRO="ubuntu20.04"
           PKG_FORMAT="deb"
           ;;
-        "ami"* )
-          DISTRO="el6"
+        "amzn"* )
+          DISTRO="el7"
           PKG_VERSION="${AEROSPIKE_C_VERSION//-/_}-1"
           PKG_FORMAT="rpm"
           ;;

--- a/scripts/os_version
+++ b/scripts/os_version
@@ -158,8 +158,12 @@ main() {
 		distro_short="${distro_id}${distro_version}"
 		;;
 	'amzn' )
-		distro_long="ami"
-		distro_short="ami"
+		case "$distro_version" in
+		201?.?? )
+			distro_version="1"
+		esac
+		distro_long="amazon_linux:${distro_version}"
+		distro_short="amzn${distro_version}"
 		;;
 	* )
 		if [ "$ID_LIKE" ]

